packages/connector-mock-social/src/constant.ts

```diff
@@ -21,4 +21,5 @@ export const defaultMetadata: ConnectorMetadata = {
   },
   readme: './README.md',
   configTemplate: './docs/config-template.json',
+  isStandard: true,
```

Comment:
As far as I know, social connectors don't support "standard"?

Reply:
Good, so let's add a mock-standard-connector later for IT use.